### PR TITLE
Update to anguar 12

### PR DESCRIPTION
**Pull request by tsiegleauq**
Updates the whole application
Delte various unused files

- gitmodules for OS3+-AU
- unused dockerignore in client/
- actions.md (was unmaintained)
- os3 proxy conf

### CODE REVIEW

**Comment by tsiegleauq** [comment]
seems prettier is getting less and less reliable by the day

File: client/src/app/site/login/components/login-mask/login-mask.component.ts [hunk 1]
```diff
@@ -134,11 +134,9 @@ export class LoginMaskComponent extends BaseComponent implements OnInit, OnDestr
...
         // Maybe the operator changes and the user is logged in. If so, redirect him and boot OpenSlides.
-        this.operatorSubscription = this.operator.operatorUpdatedEvent.subscribe(user => {
-            if (user) {
-                this.clearOperatorSubscription();
-                this.authService.redirectUser(this.currentMeetingId);
-            }
+        this.operatorSubscription = this.operator.operatorUpdatedEvent.subscribe(() => {
+            this.clearOperatorSubscription();
+            this.authService.redirectUser(this.currentMeetingId);
         });
```
Comment (GabrielInTheWorld) [inline client/src/app/site/login/components/login-mask/login-mask.component.ts:140]: Why removed the condition?

**Review by GabrielInTheWorld — APPROVED** [review]
Nice! One comment.

File: client/src/app/site/login/components/login-mask/login-mask.component.ts (hunk 1 above)
Comment (tsiegleauq) [reply client/src/app/site/login/components/login-mask/login-mask.component.ts:140]: `operator.operatorUpdatedEvent` only emitted void. eslint was detecting that and marked it as an error.

https://github.com/OpenSlides/openslides-client/blob/51aa6d5c8862a3cb2e6e9027499eb848316742e7/client/src/app/core/core-services/operator.service.ts#L116

Comment (GabrielInTheWorld) [reply client/src/app/site/login/components/login-mask/login-mask.component.ts:140]: Okay.

**Pull request merged**